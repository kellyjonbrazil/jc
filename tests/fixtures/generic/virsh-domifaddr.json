[{"name":"vnet0","mac_address":"01:23:45:67:89:ab","protocol":"ipv4","address":"192.168.122.100/24"}]
