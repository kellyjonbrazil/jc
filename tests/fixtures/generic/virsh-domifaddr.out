 Name       MAC address          Protocol     Address
-------------------------------------------------------------------------------
 vnet0      01:23:45:67:89:ab    ipv4         192.168.122.100/24
